apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/PublisherClient.cs

```diff
@@ -274,6 +305,8 @@ private static async Task<PublisherClient> CreateMaybeAsync(TopicName topicName,
                     Settings = clientCreationSettings?.PublisherServiceApiSettings,
                     ChannelOptions = grpcChannelOptions
                 };
+                clientCreationSettings?.ConfigureAction?.Invoke(builder);
+
                 var channel = isAsync ?
```

Comment:
What if what the ConfigureAction does is set a CallInvoker? Possibly a corner case but still...

Reply:
Also, I don't fully understand/remember why we need the two builders, well except for the shutdown of the channels. But if it's just for the shutdown of the channels then the comment in 298 is innacurate?
And also, for I think we don't need the second builder for shutdown now as there's the `LastCreateChannel` or similar property now?